src/test/java/com/ui_init_setup/practiceproject/tests/SwagLabsLoginTest.java

```diff
@@ -1,28 +1,21 @@
 package com.ui_init_setup.practiceproject.tests;
 
 import com.ui_init_setup.practiceproject.constant.BrowserHttpConstant;
-import com.ui_init_setup.practiceproject.junit.annotations.WIP;
-import com.ui_init_setup.practiceproject.webDriver.SupportedWebDriverProvider;
-import com.ui_init_setup.practiceproject.webDriver.ThreadLocalDriver;
-
+import com.ui_init_setup.practiceproject.constant.ErrorMessageConstant;
+import com.ui_init_setup.practiceproject.junit.extension.WebDriverSetup;
 import com.ui_init_setup.practiceproject.pages.SwagLabsLoginPage;
-import org.junit.jupiter.api.BeforeEach;
+import com.ui_init_setup.practiceproject.webDriver.ThreadLocalDriver;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvFileSource;
-import org.openqa.selenium.WebDriver;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
+@ExtendWith({WebDriverSetup.class})
 class SwagLabsLoginTest {
```

Comment:
This class should come with a warning for severe bleeding of the eyes because of its formatting (or lack thereof)